life_hc/MySQL/Gangs/fn_insertGang.sqf

```diff
@@ -8,31 +8,33 @@
     Description:
     Inserts the gang into the database.
 */
-private ["_query","_queryResult","_gangMembers","_group"];
+
 params [
-    ["_ownerID",objNull,[objNull]],
-    ["_uid","",[""]],
-    ["_gangName","",[""]]
+    ["_ownerID", objNull, [objNull]],
+    ["_uid", "", [""]],
+    ["_gangName", "", [""]]
 ];
-_group = group _ownerID;
 
-if (isNull _ownerID || _uid isEqualTo "" || _gangName isEqualTo "") exitWith {}; //Fail
+private _group = group _ownerID;
+
+if (isNull _ownerID || {_uid isEqualTo ""} || {_gangName isEqualTo ""}) exitWith {};
 
 _gangName = [_gangName] call HC_fnc_mresString;
-_query = format ["SELECT id FROM gangs WHERE name='%1' AND active='1'",_gangName];
+private _query = format ["selectGangID:%1", _gangName];
 
-_queryResult = [_query,2] call HC_fnc_asyncCall;
+private _queryResult = [_query, 2] call HC_fnc_asyncCall;
 
 //Check to see if the gang name already exists.
-if (!(count _queryResult isEqualTo 0)) exitWith {
-    [1,"There is already a gang created with that name please pick another name."] remoteExecCall ["life_fnc_broadcast",_ownerID];
+if !(count _queryResult isEqualTo []) exitWith {
```

Comment:
That change was unnecessary - either continue using count and leave that 0 or change it to [] and remove count ;)

Reply:
It wasn't unnecessary, it was a mistake. Good spot!

Reply:
happens :D